docs/program/apis/ml_training_client.md

```diff
@@ -0,0 +1,149 @@
+
+## API
+
+The ML training client API supports the following methods:
+
+{{< readfile "/static/include/services/apis/ml-training-client.md" >}}
+
+### GetTrainingJob
+
+Get training job data.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `id` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the training job you wish to get data from. Retrieve this value with [`ListTrainingJobs()`](#listtrainingjobs).
+
+**Returns**:
+
+- [(TrainingJobMetadata)](https://python.viam.dev/autoapi/viam/proto/app/mltraining/index.html#viam.proto.app.mltraining.TrainingJobMetadata): Training job metadata, including status, last modified timestamp, id, and more.
+
+```python {class="line-numbers linkable-line-numbers"}
+job_metadata = await ml_training_client.get_training_job(
+    id="INSERT YOUR JOB ID")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/ml_training_client/index.html#viam.app.ml_training_client.MLTrainingClient.get_training_job).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### ListTrainingJobs
+
+Get training job data for all jobs within an organization.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `org_id` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The ID of your organization to request training job data from. Retrieve this value with the cloud management API's [`ListOrganizations()`](/program/apis/cloud/#listorganizations).
+- `training_status` [(Optional[TrainingStatus.ValueType])](https://python.viam.dev/autoapi/viam/gen/app/mltraining/v1/ml_training_pb2/index.html#viam.gen.app.mltraining.v1.ml_training_pb2.TrainingStatus): The status of training jobs you want to filter the list by.
+
+**Returns**:
+
+- [(List[TrainingJobMetadata])](https://python.viam.dev/autoapi/viam/proto/app/mltraining/index.html#viam.proto.app.mltraining.TrainingJobMetadata): A list of training job metadata, including status, last modified timestamp, id, and more.
+
+```python {class="line-numbers linkable-line-numbers"}
+jobs_metadata = await ml_training_client.list_training_jobs(
+    org_id="INSERT YOUR ORG ID")
+
+first_job_id = jobs_metadata[1].id
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/ml_training_client/index.html#viam.app.ml_training_client.MLTrainingClient.list_training_jobs).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### CancelTrainingJob
+
+Cancel the specified training job.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `id` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the training job you wish to get data from. Retrieve this value with [`ListTrainingJobs()`](#listtrainingjobs).
+
+**Returns**:
+
+- None
+
+**Raises**:
+
+- `GRPCError`: If no training job exists with the given ID.
```

Comment:
Calling out there could be an error for other reasons too. Maybe `If no training job exists with the given ID or cancelation was otherwise unsuccessful.`